File: codigos-fonte/fibonacci.py
#Fibonacci para Python

def fibonacci(termos: int):
    valores = dict(a=0, b=1, ac=0)

    for idx in range(0, termos):
        print(valores['a'])
        valores['ac'] = valores['a'] + valores['b']
        valores['a'] = valores['b']
        valores['b'] = valores['ac']

fibonacci(64)

File: codigos-fonte/fibonacci.cs
using System;

public class Fibbonacci
{
    public static void Main(string[] args)
    {
        const int number = 64;
        fibonacci(number);
    }

    protected static void fibonacci(int termos)
    {
        long[] valores = new long[] {0, 1, 0};

        for(int i = 0; i<termos; i++)
        {
            Console.WriteLine(valores[0]);
            valores[2] = valores[0] + valores[1];
            valores[0] = valores[1];
            valores[1] = valores[2];
        }
    }
}
